packages/api/app/Controllers/Http/TechnologyController.js

```diff
@@ -0,0 +1,50 @@
+/** @typedef {import('@adonisjs/framework/src/Request')} Request */
+/** @typedef {import('@adonisjs/framework/src/Response')} Response */
+
+const Technology = use('App/Models/Technology');
+
+class TechnologyController {
+
+    async index(){
+        const technologies = Technology.all();
+        
+        return technologies;
+    }
+
+    async show({params}){
+        const technologies =  await Technology.findOrFail(params.id);
```

Comment:
Se você não for fazer nada com `technologies` você pode simplesmente retornar diretamente sem usar await.

`return Technology.findOrFail(params.id);`